optaplanner-persistence/optaplanner-persistence-jsonb/src/main/java/org/optaplanner/persistence/jsonb/api/score/buildin/bendable/BendableScoreJsonbAdapter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jsonb.api.score.buildin.bendable;
+
+import org.optaplanner.core.api.score.buildin.bendable.BendableScore;
+import org.optaplanner.persistence.jsonb.api.score.AbstractScoreJsonbAdapter;
+
+@SuppressWarnings("checkstyle:javadocstyle")
```

Comment:
I have not seen this line anywhere in core or examples. Is there any particular reason why you included it?

Reply:
It's everywhere in `optaplanner-persistence-*`, mainly in ScoreAdapters and Serializers. Here's the PR that added it with explanation: https://github.com/kiegroup/optaplanner/pull/291